config/runtime.exs

```diff
@@ -155,6 +155,10 @@ if otel_state != :purge,
       ]
     )
 
+config :membrane_rtc_engine_timescaledb,
+  repo: VideoRoom.Repo,
+  do_claeanups: Application.get_env(:membrane_videoroom_demo, :store_metrics)
```

Comment:
I don't think it's the right solution to add such an option to timescaledb. The app should not be started by default if you want it to start it conditionally.
Here's my suggestion: as mentioned here - https://stackoverflow.com/questions/53348599/exclude-application-in-a-mix-project-from-startup, add `runtime: false` to the dependency in `mix.exs` and in `application.ex`, in the existing [`if`](https://github.com/membraneframework/membrane_videoroom/blob/f129620797a17e5899157a812d5d42de30e929c5/lib/videoroom/application.ex#L24) add `Application.ensure_all_started(:membrane_rtc_engine_timescaledb)` to start that app manually